.ci/ansible/Containerfile.j2

```diff
@@ -1,5 +1,7 @@
 FROM {{ ci_base | default("ghcr.io/pulp/pulp-ci-centos:" + pulp_container_tag) }}
 
+RUN yum install -y rustc cargo patchelf
```

Comment:
Well, maybe!  Like I said there needs to be a discussion first.. but I definitely have a preference.

re: this particular line, we should be able to ship pre-built packages and it wouldn't be necessary to have these packages installed on the host.  This is just to get the tests working without having to set all that up.

Reply:
Also I'm still discussing it would Ales upstream, if he manages to get it to work, it would be a little hard to justify switching.

Although I still have some concerns about complicated it is getting, architecturally.